src/lib.rs

```diff
@@ -55,7 +79,9 @@ pub fn run_program(bin_path: &str) -> (U256, VMState) {
 /// Returns the value stored at storage with key 0 and the final vm state.
 pub fn run_program_with_custom_state(bin_path: &str, mut vm: VMState) -> (U256, VMState) {
     let program = program_from_file(bin_path);
-    vm.push_frame(program, DEFAULT_INITIAL_GAS);
+    let storage = RocksDB::open(env::temp_dir()).unwrap();
+    let storage = Rc::new(RefCell::new(storage));
+    vm.push_frame(program, DEFAULT_INITIAL_GAS, storage);
```

Comment:
Doesnt this makes `run_program_in_memory` run with rocksdb?

Reply:
fixed